hazelcast/src/test/java/other/packge/MyEP.java

```diff
@@ -0,0 +1,23 @@
+package other.packge;
+
+import com.hazelcast.map.AbstractEntryProcessor;
+
+import java.util.Map;
+
+/**
+ * This test class is intentionally in its own package
+ * as Hazelcast as a special rules for loading classes
+ * from com.hazelcast.* package.
+ *
+ */
+public class MyEP extends AbstractEntryProcessor<Integer, Integer> {
```

Comment:
Do you really need a logic inside this `EntryProcessor`? If yes, can you give a more suitable name?